docker-compose.yml

```diff
@@ -73,7 +73,7 @@ services:
       - PGHOST=localhost
       - PGDATABASE=postgis
     ports:
-      - "5439:5432"
+      - "5432:5432"
```

Comment:
This was actually changed in 0e09285a41c2e08c3eeefbcad32b46cc2747a19b so that we can run this application alongside a system PostgreSQL instance without port conflicts. I think we probably want to leave this as it was.

Reply:
https://github.com/stac-utils/stac-fastapi/pull/418/commits/8ae3686872f6d851ab3afee9b97b67b14eb8325c